packages/create-svelte/templates/default/src/routes/[...notfound]/+page.svelte

```diff
@@ -0,0 +1,6 @@
+<!-- 
+This file is shown when a URL doesn't match any other routes.
+More info: https://kit.svelte.dev/docs/advanced-routing#rest-parameters-404-pages
+-->
+<h1>404</h1>
+<p>Page not found</p>
```

Comment:
See above — we really shouldn't be returning a 200 response here

```suggestion
```

Reply:
Should we just not care about this in the skeleton/default template then? I feel like people might want to have this pattern implemented almost everytime, so we would save them some keystrokes and also possibly confusion how to deal with this (when they haven't read the advanced docs section).

Reply:
I'm just not sure what the point of such a file would be? We already have `+error.svelte`. My vote would be to not worry about it

Reply:
Ok - and now since we fall back to `Not Found` the default behavior is OK, too. Will remove these files again and adjust the changeset.